packages/waku/src/lib/builder/build.ts

```diff
@@ -148,7 +155,7 @@ const analyzeEntries = async (rootDir: string, config: ResolvedConfig) => {
   });
   const clientEntryFiles = Object.fromEntries(
     Array.from(clientFileSet).map((fname, i) => [
-      `${DIST_ASSETS}/rsc${i}-${fileHashMap.get(fname)}`,
+      `${DIST_ASSETS}/rsc${i}-${fileHashMap.get(fname) || 'lib'}`, // FIXME 'lib' is a workaround to avoid `undefined`
```

Comment:
for `waku/client` and `waku/minimal/client`, which are initialized (instead of plugin adding) in L105-106.
It's rscAnalyzePlugin that is responsible for creating the file hash. feels like room for improvement.